### ISSUE TITLE
Select & Radio not working with On Air

### ISSUE THREAD

**Issue by thetableman** [zauberzeug/nicegui#2901]
### Description

Consider the following minimal example:
```py
from nicegui import ui

ui.select(['One','Two','Three'])
ui.radio(['One','Two','Three'])

ui.run(on_air=True)
```

When selecting options from `ui.select` and `ui.radio` run from a `localhost` the UI works as expected. When doing so on a On Air instance the options don't change when selected.

![2024-04-16 10-34-16(1)](https://github.com/zauberzeug/nicegui/assets/78121198/5fce61ec-f5f1-4b3c-af49-e317d85b1f0e)

**Comment by rodja** [comment]
Thanks for bringing this to our attention @thetableman. We just fixed a small glitch on our On Air server infrastructure. Should work now. This also uncovered the lack of On Air support for `app.storage.tab` which will be fixed in the next release with #2902.

**Comment by falkoschindler** [comment]
Ok, I think we can close this issue as resolved.
